src/zc/table/interfaces.py

```diff
@@ -107,8 +109,8 @@ class IFormatter(interface.Interface):
 
     annotations = schema.Dict(
         title=u"Annotations",
-        description=
-        u"""Stores arbitrary application data under package-unique keys.
+        description=u"""\
+        Stores arbitrary application data under package-unique keys.
```

Comment:
Doesn't this add a bunch of spaces to the beginning of the description field?  I wonder if

```python
            description=(
                u"""...
                """),
```

might work better.  (Except for the extra indent level introducing noise to the diff and maybe forcing additional text rewrapping, ugh.  I imagine flake8 would object to no extra indentation after an open paren.)

Reply:
@mgedmin your proposal would create a bunch of spaces at the beginning of every line. I propose to reformat the description to match the other descriptions in this file.